BookingsAPI/Bookings.AcceptanceTests/Steps/PersonsSteps.cs

```diff
@@ -68,5 +70,22 @@ public void ThenPersonsDetailsShouldBeRetrieved()
             actual.Title.Should().Be(expected.Title);
             actual.Username.Should().Be(expected.Username);
         }
+
+        [Then(@"suitability answers for '(.*)' should be updated")]
+        public void ThenSuitabilityAnswersForShouldBeUpdated(string participant)
+        {
+            var username = _acTestContext.Participants.FirstOrDefault(x => x.UserRoleName.Equals(participant)).Username;
+            var particpantId = _acTestContext.Participants.FirstOrDefault(x => x.UserRoleName.Equals(participant)).Id;
+            _acTestContext.Request = _acTestContext.Get(_endpoints.GetPersonSuitabilityAnswers(username));
+            _acTestContext.Response = _acTestContext.Client().Execute(_acTestContext.Request);
+            if (_acTestContext.Response.Content != null)
+                _acTestContext.Json = _acTestContext.Response.Content;
+            _acTestContext.Response.StatusCode.Should().Be(HttpStatusCode.OK);
+            var model = ApiRequestHelper.DeserialiseSnakeCaseJsonToResponse<List<PersonSuitabilityAnswerResponse>>(_acTestContext.Json);
+            model[0].Answers.Count().Should().Be(2);
+            model[0].HearingId.Should().Be(_acTestContext.HearingId);
+            model[0].ParticipantId.Should().Be(particpantId);
```

Comment:
Would it make sense to test and updated value of an answer as well?

Reply:
This has been implemented.